### DISCUSSION TITLE
How to validate multiple properties in a function

### DISCUSSION THREAD

**Discussion by titanjer** [sensedeep/dynamodb-onetable#172] [Q&A] [answered]
Hi all,

I used a validate function to check the attribute values in v1.8. However, I found this is deprecated and only supports the RegExp from v1.9. Would you suggest any ideas about how to validate related properties by JS code in current implementation?

[WARNING: validation functions are DEPRECATED and will be removed soon.](https://github.com/sensedeep/dynamodb-onetable/blob/69da95d7792cdab73deaa3721cef79f5a26c6785/src/Model.js#L1290)

Thanks

**Comment by titanjer** [comment]
Currently, we inherit the Table and Model class to hook a validate function before calling super methods. This is super dirty. I hope there is better clean way to do this. lol

**Reply by mobsense** [reply]
We've posted a discussion starter about this at: #173.

I'd love to get your thoughts.

**Comment by mobsense** [comment, marked as answer]
Marking this discussion as answered and deferred to #173.
